mscore/musescore.cpp

```diff
@@ -2580,25 +2580,25 @@ void MuseScore::askResetOldScorePositions(Score* score)
             if (resPref == "No")
                   return;
             else if (resPref == "Yes")
-                  score->cmdResetAllPositions();
+                  score->cmdResetAllPositions(false);
             else { // either set to "Ask" or not at all
                   QMessageBox msgBox;
                   QCheckBox ask;
-                  ask.setText(tr("Don't ask me again."));
+                  ask.setText(tr("Remember my choice and don't ask again."));
```

Comment:
The standard convention for checkbox text is not to have any punctuation at the end, so I would remove the “.”.

(I know that the “.” was already there prior to this PR, but it shouldn't have been, so we might as well take this opportunity to fix it.)

Reply:
fine by me